python/composio/tools/env/host/workspace.py

```diff
@@ -72,6 +72,8 @@ class HostWorkspace(Workspace):
     _browsers: t.Optional[Browsers] = None
     _filemanagers: t.Optional[FileManagers] = None
 
+    _is_ssh_client_set_up: bool
```

Comment:
Consider adding type annotation initialization and documentation for this class variable:
```python
# Flag to track if SSH client has been initialized
_is_ssh_client_set_up: bool = False
```